en/topic-dates.md

```diff
@@ -73,7 +73,7 @@ before the main content -->
 <div class="center" markdown="1">
 {{number_of_events}} indexed events in {{number_of_months}} months <!-- {{mentions | size}} events including duplicates -->
 
-<!-- TODO: uncomment after January 2020 entries added: [2018](#d2018-12), [2019](#d2019-12) -->
```

Comment:
I realize we have 2020 items at the top, so optional, but 2020 in the anchor list perhaps